content/cli/orch_cli/deployments.md

```diff
@@ -500,6 +500,192 @@ Delete labels from a specific deployment.
 labels associated with this key will be deleted from the deployment
 
 
+### schedule
+
+A deployment schedule, a.k.a. execution schedule, allows the user to set a 
+scheduling rule for the execution of a particular workflow, such as install 
+or uninstall, on a deployment. This allows both single-use, i.e. postponed 
+executions, and recurrent executions (e.g. running each Sunday at 19:30).
+
+
+#### schedule list
+
+##### Usage
+
+`cfy deployments schedule list [OPTIONS] [DEPLOYMENT_ID]` 
+
+List all deployment schedules on the manager. If `DEPLOYMENT_ID` is provided, list only schedules of this deployment.
+
+##### Optional flags
+All the regular optional flags available for `list` commands, in addition to:
+
+*  `-s, --since TEXT`   -  List only schedules which have occurrences after this time. Supported formats: 
+                           `YYYY-MM-DD HH:MM`, `HH:MM`, or a relative time expression such as `+2 weeks` or `+1day+10min`.
+*  `-u, --until TEXT`   -  List only schedules which have occurrences before this time. Supported formats: 
+                           `YYYY-MM-DD HH:MM`, `HH:MM`, or a relative time expression such as `+2 weeks` or `+1day+10min`.
+*  `--tz TEXT`          -  The timezone to be used for scheduling, e.g. `EST` or `Asia/Jerusalem`. By default, the local timezone will be used. 
+                           Supports any timezone in the [tz database](en.wikipedia.org/wiki/List_of_tz_database_time_zones)
+
+Valid relative time expressions are of the form `+<integer> minutes|hours|days|weeks|months|years` or a concatenation of several such, e.g. `+1 year +3 months`. 
+These can be also written without a space after the number, without the final `s`, or using the short forms `min|h|d|w|mo|y`. 
+
+
+#### schedule get
+
+##### Usage
+
+`cfy deployments schedule get [OPTIONS] NAME`
+
+Retrieve information for a specific deployment schedule.
+
+`NAME` is the name of the deployment schedule.
+
+##### Optional flags
+All the regular optional flags available for `list` commands, in addition to:
+
+*  `--preview INTEGER`   -  Preview the N next dates for the workflow execution to run.
+
+
+#### schedule create
+
+##### Usage
+
+`cfy deployments schedule create [OPTIONS] WORKFLOW_ID`
+
+Schedule the execution of a workflow on a given deployment.
+
+`WORKFLOW_ID` is the name of the workflow.
+
+##### Mandatory flags
+
+*  `-d, --deployment-id TEXT`   -  The unique identifier for the deployment.
```

Comment:
The deployment ID should be specified as an argument in the command. I.e. `cfy deployments schedule create [OPTIONS] WORKFLOW_ID DEPLOYMENT_ID`.